packages/vuetify/src/components/VFileInput/VFileInput.ts

```diff
@@ -70,6 +70,7 @@ export default VTextField.extend({
         return typeof val === 'object' || Array.isArray(val)
       },
     } as PropValidator<File | File[]>,
+    multiple: Boolean,
```

Comment:
Defining this here breaks text-field propagating the **multiple** attribute to the input. https://github.com/vuetifyjs/vuetify/blob/0f3fa6a9204688abe23996d2df2be665f52a7e18/packages/vuetify/src/components/VTextField/VTextField.ts#L363

Reply:
Ah, I see. At a glance it is not at all clear why it has to be done with `$attrs` instead of an idiomatic prop. So let's add an explanatory comment in the `isMultiple` computed prop.